diff -wru crypto/pkcs7/pk7_enc.c crypto/pkcs7/pk7_enc.c
--- crypto/pkcs7/pk7_enc.c	2019-04-20 23:36:14.669943335 +0200
+++ crypto/pkcs7/pk7_enc.c	2019-04-20 23:37:17.020711812 +0200
@@ -17,7 +17,7 @@
 PKCS7_in_bio(PKCS7 *p7, BIO *in);
 PKCS7_out_bio(PKCS7 *p7, BIO *out);
 
-PKCS7_add_signer(PKCS7 *p7, X509 *cert, EVP_PKEY *key);
+PKCS7_add_signer(PKCS7 *p7, PKCS7_SIGNER_INFO *p7i);
 PKCS7_cipher(PKCS7 *p7, EVP_CIPHER *cipher);
 
 PKCS7_Init(PKCS7 *p7);
Only in 3rdparty/openssl: openssl_orig
