--- 3rdparty/openssl/include/openssl/e_os2.h	2020-09-22 14:55:07.000000000 +0200
+++ 3rdparty/openssl/include/openssl/e_os2.h	2021-08-18 16:06:49.361591213 +0200
@@ -237,10 +237,10 @@
 typedef UINT32 uint32_t;
 typedef INT64 int64_t;
 typedef UINT64 uint64_t;
-# elif (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L) || \
+# elif ((defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L) || \
      defined(__osf__) || defined(__sgi) || defined(__hpux) || \
-     defined(OPENSSL_SYS_VMS) || defined (__OpenBSD__)
-#  include <inttypes.h>
+     defined(OPENSSL_SYS_VMS) || defined (__OpenBSD__)) && !defined(__BORLANDC__)
+#  include <stdint.h>
 # elif defined(_MSC_VER) && _MSC_VER<1600
 /*
  * minimally required typdefs for systems not supporting inttypes.h or
